node/resolvers/index.ts

```diff
@@ -154,6 +166,63 @@ const defaultHeaders = (authToken: string) => ({
   'Proxy-Authorization': authToken,
 })
 
+// checks and configure the OrderForm based on quoteId
+const checkAndCreateQuotesConfig = async (ctx: Context): Promise<any> => {
+  const {
+    clients: { apps, checkout },
+    vtex: { logger },
+  } = ctx
+
+  const app: string = getAppId()
+  const accountSettings = await apps.getAppSettings(app)
+
+  if (!accountSettings?.hasQuoteId) {
+    const checkoutConfig: any = await checkout
+      .getOrderFormConfiguration()
+      .catch((error) => {
+        logger.error({
+          message: 'getOrderformConfiguration-error',
+          error,
+        })
+      })
+
+    if (
+      checkoutConfig?.apps.findIndex(
+        (currApp: any) => currApp.id === CHECKOUT_APP
+      ) === -1
+    ) {
+      checkoutConfig.apps.push({
+        major: 1,
+        id: CHECKOUT_APP,
+        fields: ['quoteId'],
+      })
+      const setCheckoutConfig: any = await checkout
+        .setOrderFormConfiguration(checkoutConfig, ctx.vtex.authToken)
```

Comment:
(see above comment on destructuring the authToken from the ctx)
```suggestion
        .setOrderFormConfiguration(checkoutConfig, authToken)
```